packages/cli/src/commands/redis/stats-reset.ts

```diff
@@ -0,0 +1,37 @@
+import {Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import redisApi, {RedisApiResponse} from '../../lib/redis/api'
+import confirmApp from '../../lib/apps/confirm-app'
+import heredoc from 'tsheredoc'
+import color from '@heroku-cli/color'
+
+export default class StatsReset extends Command {
+  static topic = 'redis'
+  static description = 'reset all stats covered by RESETSTAT (https://redis.io/commands/config-resetstat)'
+  static flags = {
+    app: flags.app({required: true}),
+    remote: flags.remote(),
+    confirm: flags.string({char: 'c'}),
+  }
+
+  static args = {
+    database: Args.string(),
+  }
+
+  public async run(): Promise<void> {
+    const {flags, args} = await this.parse(StatsReset)
+    const {app, confirm} = flags
+    const {database} = args
+    const addon = await redisApi(app, database, false, this.heroku).getRedisAddon()
+    const warning = heredoc(`
+      WARNING: Irreversible action.
+      All stats covered by RESETSTAT will be reset on ${color.addon(addon.name || '')}.
+    `)
+    await confirmApp(app, confirm, warning)
+
+    ux.action.start(`Reseting stats on ${color.addon(addon.name || '')}`)
```

Comment:
A great idea to introduce `ux.action.start` here, but shouldn't `Reseting` be written `Resetting`?

Reply:
lol, yup, good catch